README.md

```diff
@@ -118,6 +118,33 @@ In this case, the amount of time the retrier will wait between attempts depends
 
 The second argument to the `roko.Exponential()` method is a constant adjustment - roko will add this number to the calculated exponent.
 
+### Manual interval strategy
+
+Sometimes you only know the desired interval after each try, e.g. a rate-limited API may include a `Retry-After` header. For these cases, the manual strategy can be used, with another strategy as its default:
+
+```Go
+// manually specify interval during each try, defaulting to 10 seconds
+manual := roko.NewManual(roko.Constant(10 * time.Second))
+
+roko.NewRetrier(
+  roko.WithStrategy(manual.Register()),
+  roko.WithMaxAttempts(10),
+).Do(func(r *roko.Retrier) error {
+
+  response := apiCall() // may be rate limited
+
+  if err := response.HTTPError(); err != nil {
+    if response.Status == HttpTooManyRequests {
+      if wait, err := strconv.Atoi(response.Header("Retry-After")); err != nil {
+        r.SetNextInterval(r, wait * time.Second) // respect the API
```

Comment:
```suggestion
        manual.SetNextInterval(r, wait * time.Second) // respect the API
```

Reply:
It seems a bit unwieldy to have to reference both the strategy and the retrier to do this. I also don't like that the strategy lives outside the scope of the retrier.